src/ui/Label/Label.tsx

```diff
@@ -9,6 +9,8 @@ const label = cva(
       variant: {
         default: 'border-current',
         subtle: 'border-ds-border-line text-ds-gray-senary bg-ds-gray-primary',
+        plain:
+          'ml-2 py-0.5 border-ds-gray-tertiary text-ds-gray-senary dark:bg-ds-gray-tertiary dark:text-ds-secondary-text',
```

Comment:
Just a general question; what do we think about having margin and padding baked into a component? My gut tells me it should be done ad-hoc when we're instantiating the component and placing it on the DOM but idk

Reply:
Yeah good question. I agree with you - pulled out margin to get passed at use instead of baked into this variant. Padding is more inherent to the intended look so left it there

Reply:
totally fair! Im aligned with keeping padding too after thinking about it again